### PR TITLE
Add nodejs-redhat-cloud-services-frontend-components package

### PR DESCRIPTION
**Pull request by laviro**

### CODE REVIEW

**Comment by laviro** [comment]
@theforeman/packaging can we get this in? :)
also opened a PR to `rpm/2.3` which is also green: https://github.com/theforeman/foreman-packaging/pull/6157

**Comment by evgeni** [comment]
@laviro you can, just not over Christmas ;-)

File: rel-eng/tito.props [hunk 1]
```diff
@@ -85,6 +85,7 @@ whitelist = foreman
   nodejs-react-is
   nodejs-react-lifecycles-compat
   nodejs-react-redux
+  nodejs-redhat-cloud-services-frontend-components
```
Comment (evgeni) [inline rel-eng/tito.props:88]: please also add to `foreman-nightly-el8`

File: comps/comps-foreman-rhel7.xml [hunk 2]
```diff
@@ -34,6 +34,7 @@
       <packagereq type="default">foreman-service</packagereq>
       <packagereq type="default">foreman-telemetry</packagereq>
       <packagereq type="default">foreman-vmware</packagereq>
+      <packagereq type="default">nodejs-redhat-cloud-services-frontend-components</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-rhel7.xml:37]:
```suggestion
      <packagereq type="default">tfm-nodejs-redhat-cloud-services-frontend-components</packagereq>
```
(and then please sort again)

`nodejs-redhat-cloud-services-frontend-components` needs to be added to `comps-foreman-el8.xml`

**Review by evgeni — COMMENTED** [review]
please add el8 bits, then it's good to go

**Review by evgeni — CHANGES_REQUESTED** [review]
see previous comments

**Comment by laviro** [comment]
Thanks @evgeni !! updated

**Review by evgeni — APPROVED** [review]

**Pull request merged**